### PR TITLE
Add residential property tribunal decisions schema

### PR DESCRIPTION
**Pull request by elenatanasoiu**
The Residential Property Tribunal Judiciary publishes decisions that describe the outcome of a residential property tribunal case. This commit adds the schema for such a decision. 

For: https://trello.com/c/qzSMAFHZ/6-3-moj-residential-tribunals-finder

### CODE REVIEW

File: config/schema/elasticsearch_types/residential_property_tribunal_decision.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "fields": [
+    "tribunal_decision_category",
+    "tribunal_decision_category_name",
+    "tribunal_decision_decision_date"
```
Comment (h-lame) [inline config/schema/elasticsearch_types/residential_property_tribunal_decision.json]: We don't have a `tribunal_decision_category_name` field, but we do have a `tribunal_decision_sub_category`.  We should also check with @dwhenry and @suzannehamilton if we need to add the `expanded_search_result_fields` attribute here to define how to map the parameterised values in `tribunal_decision_category` and `tribunal_decision_sub_category` into human readable values.  For example like we have in https://github.com/alphagov/rummager/blob/master/config/schema/elasticsearch_types/aaib_report.json (not sure why we don't have it in https://github.com/alphagov/rummager/blob/master/config/schema/elasticsearch_types/employment_appeal_tribunal_decision.json though).  You can rip these values from the finder definition in https://github.com/alphagov/specialist-publisher/pull/1136)

File: config/schema/indexes/govuk.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "service_manual_guide",
     "service_manual_topic",
     "service_standard_report",
+    "residential_property_tribunal_decision",
```
Comment (suzannehamilton) [inline config/schema/indexes/govuk.json]: This should be in alphabetical order or @brenetic will be sad.

File: config/schema/elasticsearch_types/residential_property_tribunal_decision.json (hunk 1 above)
Comment (suzannehamilton) [reply config/schema/elasticsearch_types/residential_property_tribunal_decision.json]: I'm not actually sure where this is used. According to the [docs](https://github.com/alphagov/rummager/blob/master/doc/schemas.md#elasticsearch-document-types), `expanded_search_result_fields` is deprecated, and the labels are also defined in the [finder config](https://github.com/alphagov/specialist-publisher/blob/118db285035f46048063af5085da1f76b46261f1/lib/documents/schemas/aaib_reports.json). But @boffbowsh [had to update one recently](https://github.com/alphagov/rummager/pull/1150) so this information must be used somewhere.

Paul, do you know any more about this?

File: config/schema/indexes/govuk.json (hunk 2 above)
Comment (brenetic) [reply config/schema/indexes/govuk.json]: 👍 😢

File: config/schema/elasticsearch_types/residential_property_tribunal_decision.json (hunk 1 above)
Comment (boffbowsh) [reply config/schema/elasticsearch_types/residential_property_tribunal_decision.json]: I'm afraid I don't, I was (perhaps blindly) copying prior art.

File: config/schema/indexes/govuk.json (hunk 2 above)
Comment (elenatanasoiu) [reply config/schema/indexes/govuk.json]: Had alphabetical blindness there. Fixed now.

File: config/schema/elasticsearch_types/residential_property_tribunal_decision.json (hunk 1 above)
Comment (elenatanasoiu) [reply config/schema/elasticsearch_types/residential_property_tribunal_decision.json]: I'll leave it out then

Comment (elenatanasoiu) [reply config/schema/elasticsearch_types/residential_property_tribunal_decision.json]: Thanks guys!

**Review by h-lame — APPROVED** [review]

**Pull request merged**